colossus/src/main/scala/colossus/core/Worker.scala

```diff
@@ -189,6 +189,7 @@ private[colossus] class Worker(config: WorkerConfig) extends Actor with ActorMet
       }
       if (timedOut.size > 0) {
         log.debug(s"Terminated ${timedOut.size} idle connections")
+        //TODO: Tick Metric
```

Comment:
Both of these comments are for those issues we found when testing the 304 service, right?